### PR TITLE
Update Free Basics to the latest Molo version

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@saeedmb8 Ready for review cc @miltontony

**Comment by miltontony** [comment]
This looks good. @saeedmb8 can you give this a once over - there might be 1 or 2 things I'm missing.

Thanks @Ntu2koTrevor 🍰

File: freebasics/templates/core/tags/topic_of_the_day.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{% raw %}
```
Comment (smarzban) [inline freebasics/templates/core/tags/topic_of_the_day.html]: @Ntu2koTrevor can you please remove the `{% raw %}` tag

File: freebasics/templates/core/tags/topic_of_the_day.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                    {% if article.image %}
+                        <a href="{% pageurl article %}">{% image article.image width-60 %}</a>
+                    {% endif %}
+                    <h3 class="ListItem-title"><a class="ListItem ListItem--rightIcon" href="{% pageurl article %}">{{ article.title }}</a></h3>
+                    <p class="ListItem-desc">{{ article.subtitle }}</p>
+                </li>
+            </ul>
+        {% endfor %}
+{% endif %}
+{% endraw %}
```
Comment (smarzban) [inline freebasics/templates/core/tags/topic_of_the_day.html]: Here as well

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,7 @@
-molo.core==3.4.2
+molo.core==3.12.0
 molo.profiles==1.0.0
```
Comment (smarzban) [inline requirements.txt]: @Ntu2koTrevor can you update the profile version to latest as well, then FED can update the CSS for password recovery

File: requirements.txt [hunk 4]
```diff
@@ -1,6 +1,7 @@
...
 molo.commenting==0.5.4
```
Comment (smarzban) [inline requirements.txt]: @Ntu2koTrevor let's update commenting to `1.0.0` , and you need to add namespace to commenting URLs in the templates, also you need to add this URL `url(r'', include('django_comments.urls')),` to freebasics URLs as we moved it out from commenting URLs.
https://github.com/praekelt/molo.commenting#molo-commenting

File: requirements.txt [hunk 5]
```diff
@@ -1,6 +1,7 @@
...
+molo.yourwords==1.2.1
 molo.polls==2.0.2
```
Comment (smarzban) [inline requirements.txt]: @Ntu2koTrevor please pin this to `2.2.1`

**Comment by smarzban** [comment]
@Ntu2koTrevor can you please generate the EN PO file as well

**Comment by g1039** [comment]
@saeedmb8 Please review

**Comment by smarzban** [comment]
👍 🌟 🍰

**Comment by miltontony** [comment]
Thanks @Ntu2koTrevor 🍰 ⭐️

**Pull request merged**